docker-compose.yml

```diff
@@ -29,6 +29,7 @@ services:
       - "db"
     environment:
       - DYNACONF_DATABASE_URL=${FEXSERVICE_DATABASE_URL}
+      - GITHUB_TOKEN=${GITHUB_TOKEN}
```

Comment:
não seria `DYNACONF_GITHUB_TOKEN` ? o Dynaconf lê as variavéis exportadas com o prefixo `DYNACONF`

Reply:
Ficaria algo assim?
```
DYNACONF_GITHUB_TOKEN=${GITHUB_TOKEN}
```